documentation-src/build.gulp.js

```diff
@@ -36,58 +38,83 @@ var documentationRoot = path.join(projectRoot, 'documentation');
 var jsRoot = path.join(documentationRoot, 'js');
 var cssRoot = path.join(documentationRoot, 'css');
 
-
 var customMarkedRenderer = new marked.Renderer();
 var oldHeadingRenderer = customMarkedRenderer.heading;
 customMarkedRenderer.heading = function(text, level) {
-  if(level > 3) {
+  if (level > 3) {
     return '<h' + level + '>' + text + '</h' + level + '>';
   }
   return oldHeadingRenderer.apply(this, arguments);
 };
+
+var header = a.communityHeader({
+  menu:{
+    project: {
+      label: "algoliasearch-helper",
+      url: "/algoliasearch-helper-js/"
+    }
+  },
+  sideMenu: [
+    { name: "Getting started", dropdownItems: null, url: "/algoliasearch-helper-js/gettingstarted.html" },
```

Comment:
You should use relative path to the files. It will help local deployment (and not break on surge / staging)